src/components/AutoDeposit/FrequentlyAskedQuestions.vue

```diff
@@ -0,0 +1,78 @@
+<template>
```

Comment:
This is a separate component, almost identical to the MG FAQ component. Future improvement would be to make these 2 components get content from contentful based on a prop key.

Reply:
Yea we should make an FAQ component someday!